src/main/java/eu/europeana/metis/sandbox/config/SandboxConfig.java

```diff
@@ -76,4 +88,32 @@ String initialQueue() {
   ZipFileReader zipFileReader() {
     return new ZipFileReader();
   }
+
+  @Bean
+  ClasspathResourceResolver lsResourceResolver() {
+    return new ClasspathResourceResolver();
+  }
+
+  @Bean
+  SchemaProvider schemaProvider() throws IOException {
+    return new SchemaProvider(PredefinedSchemasGenerator.generate(schemaProperties()));
+  }
+
+  private Properties schemaProperties() throws IOException {
+    PropertiesFactoryBean propertiesFactoryBean = new PropertiesFactoryBean();
+    propertiesFactoryBean.setLocation(new ClassPathResource("predefined-schemas.properties"));
+    propertiesFactoryBean.afterPropertiesSet();
+    return propertiesFactoryBean.getObject();
+  }
```

Comment:
I guess this may be impacted by another remark I made. The question is: how do we handle workflow step-specific configuration like the validation schema information? We don't want separate property files for all steps, do we? Is it a good idea to integrate that into the application.yml? And if this implies a change in the validation library in Metis, that's fine.

Reply:
I found a way to keep the config in the yml file, at first the reason I had to separate was that I did not found a way to have all together and provide the `Properties` to the `PredefinedSchemasGenerator.generate` method.
I will upload the way I found for you to check on it